frontend/modules/risks/risks.sagas.ts

```diff
@@ -639,6 +642,38 @@ export function* attachLinkResources({ links }) {
 	}
 }
 
+function* fetchMitigationCriteria({teamspace}) {
+	try {
+		const mitigationCriteriaTeamspace = yield select(selectMitigationCriteriaTeamspace);
+		if (mitigationCriteriaTeamspace !== teamspace) {
```

Comment:
what is mitigationCriteriaTeamspace and can it ever differ to teamspace that is being passed? @m1chalkubiak 

or does this only happens if someone changes model they're viewing?

Reply:
@carmenfan So, as mitigation criteria are directly connected to the teamspaces, I assumed that for some cases there won't be any reasons to fetch it again. In this meaning, the mitigationCriteriaTeamspace is a sort of meta-data which inform for which workspace these mitigation criteria are. Of course, in case of lack of it, we simply fetch it.

Reply:
So what you mean is you'd cache the last result and if the teamspace is the same you don't bother fetching?

I think this logic is not necessary. Mitigation criteria should always be fetched once per model load to ensure it's up to date.